documentation.html

```diff
@@ -631,6 +631,41 @@ <h2 class="doc-header">Image</h2>
             </div>
             <hr>
         </section>
+        <section class="input-component" id="input">
+            <div class="doc-header-container">
+                <h2 class="doc-header">Input</h2>
+            </div>
+            <div class="doc-body-container">
+                <p class="p3">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Maiores ratione animi minima quos modi provident odio ducimus necessitatibus, voluptatem, autem impedit saepe eaque sapiente quis reiciendis nam tempora minus? Libero?</p>
+                <br>
+                <form class="form-action" action="">
+                    <label class="input-label" for="username">User Name :</label>
+                    <input class="input-action" type="text" id="userName" name="username" placeholder="Enter user name..">
+                    <label class="input-label" for="username">User Name :</label>
+                    <input class="input-action input-danger border-danger" type="text" id="userName" name="username" placeholder="Enter user name..">
```

Comment:
Is it just for view? Valid or invalid. Aren't you really checking it?

Reply:
Yes . for now it is only static css. In project it will be use as dynamic.